src/corehost/Windows/gen-buildsys-win.bat

```diff
@@ -10,8 +10,6 @@ if %1=="/?" GOTO :USAGE
 
 setlocal
 set __sourceDir=%~dp0..
-:: VS 2015 is the minimum supported toolset
-set __VSString=14 2015
```

Comment:
Since the second arg isn't used as intended, do you want to clean up the arg list to remove it? Also the help text on the bottom references the second arg which should be removed.

Reply:
@steveharter I'm not sure what you mean by the second arg isn't used as intended? I think the template generator needs bot the version and year or am I missing something.

Reply:
For gen-buildsys-win.bat file, the second `in` argument is <VSVersion> which isn't used anymore. I believe the original intention was to call the bat file with a string like `vs2017', and then set the __VSString based on that. The help mentions <VSVersion>.

Reply:
I see I understand I switched to using that parameter in the corehost scripts and just moved only to 2017 for the uwp scripts because they don't work otherwise.